sljit_src/sljitNativeARM_32.c

```diff
@@ -41,6 +41,21 @@ SLJIT_API_FUNC_ATTRIBUTE const char* sljit_get_platform_name(void)
 #endif
 }
 
+#if (defined SLJIT_DETECT_NEON && SLJIT_DETECT_NEON)
+#define CPU_FEATURE_NEON	(1UL << 12)
+
+#include <sys/auxv.h>
```

Comment:
Is it available in al compilers?

Reply:
it is provided by the OS and available in most ELF systems.

this code is linux specific though and will need more care before being ready, this was mainly posted for testing and discussion